package.json

```diff
@@ -78,7 +78,8 @@
     "ShowIntercom",
     "SplitKeyboardShortcut",
     "ToolkitReports",
-    "FilterCategories"
+    "FilterCategories",
+    "CtrlEnterCleared"
```

Comment:
Like @dbaldon said, we don't really like to make features "default" unless they're pretty substantial additions.

That said, this feature _would_ be part of a feature I think should be default on which is "Keyboard Shortcuts".

Unfortunately, I've not had the time to tackle that feature I wish I did, if you'd like to take a stab at making a generic "keyboard shortcuts" feature that's easy to add more shortcuts to, feel free and we'll consider making a default-enabled feature but until then, I'll accept this feature as a one-off.